app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -79,6 +79,32 @@ def metrics_connect
                                      authentication_password('default'))
     end
 
+    def run_generic_operation(operation, path)
+      require 'hawkular_all'
+      host_port = "#{hostname}:#{port}"
+      client = ::Hawkular::Operations::OperationsClient.new(:host        => host_port,
+                                                            :credentials => {
+                                                              :username => authentication_userid('default'),
+                                                              :password => authentication_password('default')
+                                                            })
```

Comment:
I feel like this should be moved into the connect method as a service request, similar to how we do on other providers....so the caller would do:

`ems.connect(:service => "Operations")` and get the operations client back.